.github/project-community-pr-assigner.yml

```diff
@@ -125,3 +117,135 @@
 
 '**/package.json':
   - team: flux
+
+"php/blueprint":
+  - team: ghidorah
+
+"php/remote-specs-validation":
```

Comment:
I guess it should be `packages/php/remote-specs-validation` here (similar to blueprint)?